operator/charts/embedded-cluster-operator/templates/embedded-cluster-troubleshoot-goldpinger.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: embedded-cluster-troubleshoot-goldpinger
+  labels:
+    troubleshoot.sh/kind: support-bundle
+    {{- with (include "embedded-cluster-operator.labels" $ | fromYaml) }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+data:
+  support-bundle-spec: |
+    apiVersion: troubleshoot.sh/v1beta2
+    kind: SupportBundle
+    metadata:
+      name: embedded-cluster-troubleshoot-goldpinger
+      labels:
+        troubleshoot.sh/kind: support-bundle
+    spec:
+      collectors:
+      - goldpinger:
+          namespace: goldpinger
+          image: bloomberg/goldpinger:3.10.1
```

Comment:
shouldn't this go through `proxy.replicated.com/anonymous/` like all the other images? because the customer env might not allow traffic to dockerhub. i would recommend using a digest as well so the image gets cached and still works if there is a dockerhub outage.